src/helper/waylandsocketwatcher.cpp

```diff
@@ -53,31 +55,35 @@ void WaylandSocketWatcher::start()
         // Time is up and a socket was not found
         if (!m_watcher.isNull())
             m_watcher->deleteLater();
-        qWarning("Wayland socket watcher for \"%s\" timed out",
-                 qPrintable(m_socketPath));
+        qWarning("Wayland socket watcher timed out");
         m_status = Failed;
         Q_EMIT failed();
     });
 
     // Check if the socket exists
     connect(m_watcher, &QFileSystemWatcher::directoryChanged, this,
             [this](const QString &path) {
-        qDebug() << "Directory" << path << "has changed, checking for" << m_socketPath;
+        qDebug() << "Directory" << path << "has changed, checking for Wayland socket";
```

Comment:
still print `m_runtimeDir.absolutePath()` so we can diagnose when debugging.

Reply:
`QFileSystemWatcher` passes to the signal the same value it received via `addPaths`. In this case it would be `m_runtimeDir.absolutePath()`, so the change you suggested will not affect the output.

But I agree that there's not enough info for debugging. Added a message with a full path to the socket we found:
```
Dec 13 20:05:41 sddm-helper-start-wayland[1074]: Starting Wayland process "weston --shell=fullscreen-shell.so" "sddm"
Dec 13 20:05:41 sddm-helper-start-wayland[1074]: started succesfully "weston --shell=fullscreen-shell.so"
<... weston startup logs ...>
Dec 13 20:05:41 sddm-helper-start-wayland[1074]: Directory "/run/user/979" has changed, checking for Wayland socket
Dec 13 20:05:41 sddm-helper-start-wayland[1074]: Directory "/run/user/979" has changed, checking for Wayland socket
Dec 13 20:05:41 sddm-helper-start-wayland[1074]: Found Wayland socket "/run/user/979/wayland-1"
```